src/main/java/Duke.java

```diff
@@ -1,10 +1,131 @@
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+
+    public static void printBorder() {
+        String border = "____________________________________________________________ \n";
+        System.out.println(border);
+    }
+
+    public static void printGreeting(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String border = "____________________________________________________________ \n";
+        String greeting = "Hello! I'm Duke\n" + "What can I do for you?\n";
+        System.out.println(logo + border+ greeting);
+    }
+
+    public static void printExit(){
+        String exit = "Bye. Hope to see you again soon! \n";
+        System.out.println(exit);
+    }
+
+    public static void main(String[] args) {
+
+        Task[] tasks = new Task[100];
+        Integer listCount = 0;
+
+
+        printGreeting();
+        printBorder();
+
+
+        Scanner input = new Scanner(System.in);
+        String entry = input.nextLine();
+
+
+        while (!entry.equals("bye")){
+
+            String[] userInput = entry.split(" ", 2);
+
+            switch(userInput[0]){
+                case "list":
+                    printBorder();
+
+                    int counter = 1;
+                    for (Task t: tasks){
+                        if (t== null){
+                            break;
+                        }
+                        else{
+                            System.out.println(counter + ". " + t);
+                            counter++;
+                        }
```

Comment:
I think the layout for "if-else" statements would better fit the coding standards if it were to be changed to this?
```suggestion
                        if (t== null){
                            break;
                        } else {
                            System.out.println(counter + ". " + t);
                            counter++;
                        }
```